### PR TITLE
add nigeria_plateau_state spider

### LINKED ISSUE #931: nigeria_plateau_state
**Links to bulk download and/or API documentation**
https://plateaustatebpp.com/ocds

**Links to bulk download and/or API endpoints**
Looks similar to other Nigerian states.

**Link to CRM issue**
N/A

**Priority**
anytime

**Data structure**
Not checked

**Publication format**
Not checked

*If the publication passes the [basic criteria](https://standard.open-contracting.org/latest/en/getting_started/quality/#basic-criteria), please follow the steps described at the [process note](https://docs.google.com/document/d/12d61HXZaD3wBYN479ShfZmc0xW29fJvmGNhkyf4xUhg/edit#heading=h.2fhb7gu8ein9).*
Not checked

### PR DESCRIPTION
**Pull request by Ravf95**
Closes #931

### CODE REVIEW

**Comment by Ravf95** [comment]
I'm not sure why flake8 fails

**Comment by jpmckinney** [comment]
They must be due to a new version of flake8.

File: kingfisher_scrapy/spiders/nigeria_plateau_state.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import scrapy
+
+from kingfisher_scrapy.base_spiders import SimpleSpider
+
+
+class NigeriaPlateauState(SimpleSpider):
+    """
+    Domain
+      Nigeria Plateau State
```
Comment (yolile) [inline kingfisher_scrapy/spiders/nigeria_plateau_state.py:9]: We can add https://plateaustatebpp.com/ocds as the Bulk download documentation to at least have something

**Review by yolile — DISMISSED** [review]

File: kingfisher_scrapy/spiders/nigeria_plateau_state.py (hunk 1 above)
Comment (Ravf95) [reply kingfisher_scrapy/spiders/nigeria_plateau_state.py:9]: Done

Comment (yolile) [reply kingfisher_scrapy/spiders/nigeria_plateau_state.py:9]: I think you forgot to push your commit :)

Comment (Ravf95) [reply kingfisher_scrapy/spiders/nigeria_plateau_state.py:9]: Sorry

**Review by yolile — APPROVED** [review]

**Pull request merged**